tests/data/annotated_dialogues.json

```diff
@@ -4,37 +4,76 @@
         "conversation": [
             {
                 "participant": "USER",
-                "intent": "DISCLOSE.NON-DISCLOSE",
+                "dialogue_acts": [
+                    {
+                        "intent": "DISCLOSE.NON-DISCLOSE",
+                        "slot_values": []
```

Comment:
Do empty slot-values need to be explicitly specified? It'd be good to make sure in the loader that they are initialized with an empty list if this field is missing from the json file.

Reply:
It does not have to explicitly specificied. I removed it from the last two conversations but leave it for the first in case we want to use it for future tests.